vignettes/publisher_vignette.Rmd

```diff
@@ -0,0 +1,122 @@
+---
+title: "fulltext Vignette: Additional Publishers"
```

Comment:
we're not using rmarkdown here, copy the format of the other vignettes - see https://raw.githubusercontent.com/ropensci/fulltext/master/vignettes/getting_fulltext.Rmd